akita/src/entityService.ts

```diff
@@ -0,0 +1,9 @@
+import { EntityState, getEntityType, getIDType } from './types';
+import { Observable } from 'rxjs';
+
+export abstract class EntityService<S extends EntityState> {
```

Comment:
I thought this EntityService should have a getter for the resourceName and baseApi.

Reply:
Yes, I thought about it, and it's relevant only for REST APIs in general. We could always change it later.